### ISSUE TITLE
Allow using ini file to pass options to the installer

### ISSUE THREAD

**Issue by yakky** [nephila/djangocms-installer#110]
Using a ini file will make easier to create repeatable installations

**Comment by frost-nzcr4** [comment]
I've completed the ini config working task and this branch https://github.com/nephila/djangocms-installer/compare/develop...frost-nzcr4:ini-parse may be interesting for you as well. Also at work I need to get dump of config right after all parsing is completed — dump described here https://github.com/frost-nzcr4/djangocms-installer/compare/ini-parse...frost-nzcr4:ini-dump it helps me partially workaround issue #94: when install failed I re-run it with dumped config.

**Comment by yakky** [comment]
@frost-nzcr4 this is great! I want those, they look awesome and basically ready for merge (probably just adding ini format documentation)

**Comment by frost-nzcr4** [comment]
### config-file

Sample ini config with all available arguments https://github.com/frost-nzcr4/djangocms-installer/blob/ini-parse/config.ini.sample.
Run:

```ShellSession
djangocms_installer --config-file /path/to/config.ini project_name
```

General config can be cutomised with any available arguments:

```ShellSession
djangocms_installer --config-file /path/general-config.ini -p /path/other/proj -s -q project_name
```

If config.ini not contains `no-input = true` and `-q` argument isn't set then one act as a placeholder with default values for wizard.

### config-dump

Dumps all arguments to custom config.ini:

```ShellSession
djangocms_installer --config-dump /path/config.ini -p . project_name
```

if installation fails dump can be used to fix some arguments and re-run installer with dumped config:

```ShellSession
djangocms_installer --config-dump /path/config.ini --db postgres://wrong-usr:pwd@host/db -p . project_name
# failed
djangocms_installer --config-file /path/config.ini --db postgres://correct-user:pwd@host/db -p . project_name
# succeed
```

**Comment by yakky** [comment]
Fixed by #197 
A great thanks @frost-nzcr4 for contributing this exciting feature
